xcom/XCOMTest.java

```diff
@@ -0,0 +1,179 @@
+package grakn.example.xcom;
+
+import grakn.client.GraknClient;
+import graql.lang.Graql;
+import graql.lang.query.GraqlDefine;
+import graql.lang.query.GraqlGet;
+import org.junit.*;
+
+import javax.xml.stream.XMLStreamException;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+
+import static graql.lang.Graql.parse;
+import static org.junit.Assert.*;
+
+
+public class XCOMTest {
+
+	GraknClient client;
+	GraknClient.Session session;
+	String keyspaceName = "xcom_test";
+
+	@Before
+	public void loadSchema() {
+		client = new GraknClient("localhost:48555");
+		session = client.session(keyspaceName);
+		GraknClient.Transaction transaction = session.transaction().write();
+
+		try {
+			byte[] encoded = Files.readAllBytes(Paths.get("schemas/xcom-schema.gql"));
+			String query = new String(encoded, StandardCharsets.UTF_8);
+			transaction.execute((GraqlDefine) Graql.parse(query));
+			transaction.commit();
+			System.out.println("Loaded the " + keyspaceName + " schema");
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+	}
+
+	@After
+	public void deleteKeyspace() {
+		client.keyspaces().delete(keyspaceName);
+		System.out.println("Deleted the " + keyspaceName + " keyspace");
+		session.close();
+		client.close();
+	}
+
+	@Test
+	public void testMigration() throws FileNotFoundException {
+		Migration.main(new String[] { keyspaceName });
+		assertMigrationResults();
+	}
+
+	@Ignore
```

Comment:
Did this test always take too long to complete?

Reply:
I'm pretty sure this test used to succeed in a reasonable time. I don't know why it's become so slow.